Common/include/adt_structure.hpp

```diff
@@ -91,6 +91,70 @@ class su2_adtComparePointClass {
   su2_adtComparePointClass();
 };
 
+/*! 
+ * \class su2_BBoxTargetClass
+ * \brief  Class for storing the information of a possible bounding box candidate
+           during a minimum distance search.
+ * \author E. van der Weide
+ * \version 6.1.0 "Falcon"
+ */
+class su2_BBoxTargetClass {
```

Comment:
@economon, I renamed (hopefully) all the classes according to the SU2 convention and moved the su2_blas functionalities to the class CBlasStructure and similarly for the graph coloring functionality. The question is what to do what the standalone function DetermineManufacturedSolution (file data_manufactured_solutions.cpp in SU2_CFD/src). I can also put them into a class, but we have plans to completely redo the manufactured solutions anyway. So I propose to leave this one as it is for now. Do you agree?

As far as the prefix fem_ is concerned, there are a few questionable ones, namely blas_structure.cpp and graph_coloring_structure.cpp. In principle the functionality in these files is general, but it is only used for the FEM solver at the moment. Should we add the prefix fem_ there as well? 

A special case is geometry_structure_fem_part.cpp. This flle contains the additional functions I added to CPhysicalGeometry needed for the FEM solver, as well as the implementation of functions of several classes (CFaceOfElement, CBoundaryFace, CMatchingFace) needed for the partitioning of the FEM solver. Would you like to put those functions in a separate file?

Reply:
If you feel a particular file is general, then leave it without 'fem_*' for now, I would say. For any that are still ambiguous, let's also leave those, as we can treat this in a later PR when we start moving the geometry files around on the FVM side too (might require more thinking at that time). Same for MMS.

In short, since you've already put 'fem_*' for the obvious ones, let's stop there and merge. Thanks for all the changes.